### PR TITLE
Prevent import from stopping when publishing trashed content

### PR DESCRIPTION
**Pull request by htollefsen**
Aims to solve #37.

### CODE REVIEW

File: src/Transfer/EzPlatform/Repository/Manager/ContentManager.php [hunk 1]
```diff
@@ -279,4 +284,25 @@ private function assignStructFieldValues(ContentObject $object, $struct)
...
+    {
+        $query = new Query();
+        $query->filter = new Criterion\ContentId($object->getContentInfo()->id);
+        $trash = $this->repository->getTrashService()->findTrashItems($query);
+        if ($trash->count > 0) {
+            /** @var TrashItem $trashItem */
+            $trashItem = $trash->items[0];
+            $parentLocation = $this->repository->getLocationService()->loadLocation($trashItem->parentLocationId);
+            $this->repository->getTrashService()->recover($trashItem, $parentLocation);
+            if ($this->logger) {
```
Comment (htollefsen) [inline src/Transfer/EzPlatform/Repository/Manager/ContentManager.php:302]: @valisj I'm unable to inject a logger to get coverage in this `ìf`. Are you able to see my error?

File: tests/unit/Transfer/EzPlatform/Tests/Repository/ContentTreeServiceTest.php [hunk 2]
```diff
@@ -66,4 +67,24 @@ public function testCreateWithInvalidArgument()
...
+            $this->getContentObject(
+                array(
+                    'name' => 'Test folder',
+                ),
+                'folder',
+                'content_trashed_folder_0'
+            )
+        );
+
+        $this->service->create($tree);
```
Comment (ghost) [inline tests/unit/Transfer/EzPlatform/Tests/Repository/ContentTreeServiceTest.php:84]: How about `$this->service->setLogger(new \Psr\Log\NullLogger());`?

File: src/Transfer/EzPlatform/Repository/Manager/ContentManager.php (hunk 1 above)
Comment (ghost) [inline src/Transfer/EzPlatform/Repository/Manager/ContentManager.php:302]: See https://github.com/transfer-framework/ezplatform/pull/52/files#r52584206

Comment (htollefsen) [inline src/Transfer/EzPlatform/Repository/Manager/ContentManager.php:302]: I tried that accually. Same result (still not covered). Committing it temporarily to get it confirmed.

File: tests/unit/Transfer/EzPlatform/Tests/Repository/ContentTreeServiceTest.php [hunk 3]
```diff
@@ -66,4 +67,25 @@ public function testCreateWithInvalidArgument()
...
+        $tree = $this->getTreeObject(2,
+            $this->getContentObject(
+                array(
+                    'name' => 'Test folder',
+                ),
+                'folder',
+                'content_trashed_folder_0'
+            )
+        );
+
```
Comment (ghost) [inline tests/unit/Transfer/EzPlatform/Tests/Repository/ContentTreeServiceTest.php:83]: Use this for the setUp method:

```
public function setUp()
     {
         $objectService = new ObjectService(static::$repository);
         $objectService->setLogger(new NullLogger());
         $this->service = new ContentTreeService(static::$repository, $objectService);
         $this->service->setLogger(new NullLogger());
     }
```

File: src/Transfer/EzPlatform/Repository/Manager/ContentManager.php (hunk 1 above)
Comment (ghost) [inline src/Transfer/EzPlatform/Repository/Manager/ContentManager.php:302]: See https://github.com/transfer-framework/ezplatform/pull/52/files#r52589750

Comment (htollefsen) [inline src/Transfer/EzPlatform/Repository/Manager/ContentManager.php:302]: Perfect!

**Pull request merged**